.github/ISSUE_TEMPLATE/eol_collection.md

```diff
@@ -14,13 +14,27 @@ When opening PRs, add a link back to this issue for easier tracking.
 -->
 
 - [ ] Write a post to https://community.gazebosim.org/ announcing the community it has last X weeks for finishing their PRs.
+    * <!-- LINK POST HERE -->
 - [ ] Check library versions that can be EOL’d on the [versions table](https://github.com/ignitionrobotics/docs/blob/master/tools/versions.md)
     * <!-- LIST LIBRARIES HERE -->
-- [ ] Make releases with outstanding changes in each library about to be EOl'ed
-- [ ] Archive release repos for all libraries after the last release has been made
-- [ ] Update docs ([example](https://github.com/ignitionrobotics/docs/pull/124))
-- [ ] Update release-tools ([example](https://github.com/ignition-tooling/release-tools/pull/369))
-- [ ] Deprecate homebrew bottles ([example](https://github.com/osrf/homebrew-simulation/pull/1242))
+- [ ] Make releases with outstanding changes in each library about to be EOL'ed
+    * <!-- LIST PRs HERE -->
+- [ ] Archive release repos in [ignition-release](https://github.com/ignition-release) for all libraries after the last release has been made
+- [ ] Update `release-tools` after all releases have been made ([example](https://github.com/ignition-tooling/release-tools/pull/369))
+    * <!-- LINK PR HERE -->
+- [ ] Deprecate homebrew bottles (don't disable them) ([example](https://github.com/osrf/homebrew-simulation/pull/1242))
```

Comment:
we don't yet have a good example in a single pull request, because I think it has taken us 2 pull requests to get it right. I would make sure we reference both pull requests here to try to avoid this mistake in the future:

* blueprint examples: https://github.com/osrf/homebrew-simulation/pull/1242 and https://github.com/osrf/homebrew-simulation/pull/1256
* dome: https://github.com/osrf/homebrew-simulation/pull/1734 and https://github.com/osrf/homebrew-simulation/pull/1737

Reply:
What do you think? [9e444fd](https://github.com/ignition-tooling/release-tools/pull/616/commits/9e444fd0fd1aeb8af4f4ba06fc4d71fa96df71ef)